profile/E-ARK-eHealth1-ROOT.xml

```diff
@@ -210,7 +215,8 @@
                     <p xmlns="http://www.w3.org/1999/xhtml">The name of the organisation(s) that originally created the data being transferred MUST be given.</p>
                     <p xmlns="http://www.w3.org/1999/xhtml">Please note that this might be different from the organisation which has been charged with preparing and sending the SIP to the archives.</p>
                     <dl xmlns="http://www.w3.org/1999/xhtml">
-                        <dt>METS XPath</dt><dd>metsHdr/agent/name</dd>
+                    <!--the requirement text potentially suggests this path refers only to agents with a creator role, but it does not-->
+                        <dt>METS XPath</dt><dd>mets/metsHdr/agent/name</dd>

```

Comment:
This requirement is only for Creator agents, see above, the wording for EHR6 is wrong. This should be a modification of SIP12 from MAY to MUST.